lib/MySQL_Session.cpp

```diff
@@ -395,21 +419,24 @@ int MySQL_Session::handler() {
 						c=*((unsigned char *)pkt.ptr+sizeof(mysql_hdr));
 						switch ((enum_mysql_command)c) {
 							case _MYSQL_COM_QUERY:
+								__sync_add_and_fetch(&thread->status_variables.queries,1);
 #ifdef DEBUG
 								if (mysql_thread___session_debug) {
 									if ((pkt.size>9) && strncasecmp("dbg ",(const char *)pkt.ptr+sizeof(mysql_hdr)+1,4)==0) {
-										if (mysql_thread___commands_stats==true) {
-											CurrentQuery.init((unsigned char *)pkt.ptr,pkt.size,true);
-											CurrentQuery.start_time=thread->curtime;
-											CurrentQuery.query_parser_init();
-											CurrentQuery.query_parser_command_type();
-										}
+											CurrentQuery.begin((unsigned char *)pkt.ptr,pkt.size,true);
+//										if (mysql_thread___commands_stats==true) {
```

Comment:
My opinion is to get rid of such commented code. You can always find it in git history if you need it, based on the commit messages (if they are descriptive enough).